calculator/operations.py

```diff
@@ -1,23 +1,43 @@
+from __future__ import division
 
 def add(*args):
-    # your implementation here
-    pass
+        if len(args) == 1:
```

Comment:
Agree with pkkondamuri. If you use functional programming (lambda) you'd have the right answer.